src/nxdoc/nuxeo-server/installation/nuxeo-cluster-architecture/nuxeo-architecture-components/asynchronous-processing-architecture.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Asynchronous Processing
+description:
+review:
+    comment: ''
+    date: '2021-03-10'
+    status: ok
+toc: true
+tree_item_index: 700
+---
+
+Asynchronous processing relies on the **WorkManager**, **Nuxeo Stream**, **KeyValue Store** and the **PubSub Service**.
+
+## Concepts
+
+When running in cluster mode, the Nuxeo nodes need to communicate so the following services work in a distributed way:
+
+- The **[WorkManager]({{page page='work-and-workmanager'}})** can distribute its Works among nodes and share a common state.
+- **[Nuxeo Stream]({{page page='nuxeo-stream'}})** and the **[Bulk Service]({{page page='bulk-action-framework'}})** distribute processing among nodes and handle failover.
+- The **KeyValue Store** enabling a shared [Transient Store]({{page page='transient-store'}}).
```

Comment:
The **KeyValue Store** is shared by all nodes and used by different services like the "Transient Store"